src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -407,9 +407,9 @@ class IndexLogEntryTest extends SparkFunSuite with SQLHelper with BeforeAndAfter
   test("Directory.fromDirectory and fromLeafFiles where files belong to multiple" +
     "subdirectories.") {
     // File Structure
-    // testDir/a/f1
-    // testDir/a/b/f2
-    // testDir/a/c/f3
+    // testDir/temp/a/f1
+    // testDir/temp/a/b/f2
+    // testDir/temp/a/c/f3
```

Comment:
This change is an resolution of [this comment](https://github.com/microsoft/hyperspace/pull/162/files#r489593365). It's repeated across many other tests.